promptsource/promptsource.py

```diff
@@ -103,11 +105,11 @@ def show_text(t, width=WIDTH):
     all_infos = manager.dict()
     all_datasets = list(set([t[0] for t in template_collection.keys]))
 
-    def get_infos(d_name):
+    def get_infos(d_name, all_infos):
         all_infos[d_name] = get_dataset_infos(d_name)
 
-    pool = Pool(processes=len(all_datasets))
-    pool.map(get_infos, all_datasets)
+    pool = ThreadPool(processes=len(all_datasets))
```

Comment:
A safer choice on recent macOS is to replace `fork` with `spawn`. Unfortunately, we can't really change `Pool`'s start method to `spawn` without breaking streamlit. So the only convenient choice is `ThreadPool` (which is also good for I/O bound in this case).